File: sicpa-root/sicpa-dto/src/main/java/com/sicpa/demo/dto/EmployeeDto.java
package com.sicpa.demo.dto;

import java.io.Serializable;

import javax.validation.constraints.NotEmpty;
import javax.validation.constraints.NotNull;

import lombok.Data;
import lombok.EqualsAndHashCode;

@Data
@EqualsAndHashCode(callSuper = true)
public class EmployeeDto extends BaseDto implements Serializable {
	private static final long serialVersionUID = 1L;

	private Long id;
	@NotEmpty
	private String name;
	@NotEmpty
	private String surname;
	@NotNull
	private Integer age;
	//@Email
	private String email;
	private String position;
	//private List<DepartmentEmployeeDto> departmentEmployees;
	//private List<UserDto> users;

}

File: sicpa-web/src/app/models/employee-dto.ts
import {GenericDto} from "./generic-dto";

export class EmployeeDto extends GenericDto {
  id: number;
  name: string;
  surname: string;
  lastname: string
  age: number;
  email: string;
  position: string;

  constructor() {
    super();
  }
}
